iothub/service/src/DirectMethod/Models/DirectMethodServiceRequest.cs

```diff
@@ -12,13 +12,17 @@ namespace Microsoft.Azure.Devices
     /// </summary>
     public class DirectMethodServiceRequest
     {
-        private object _payload;
-
         /// <summary>
-        /// Initialize an instance of this class.
+        /// Creates an instance of this class with the specified payload to be serialized.
```

Comment:
I converted this file as close as I could to what we had before, meaning we take their object and serialize it and stuff it in a JSON type. From there we can serialize it (again!?) or offer it back to the user if they want (but why would they want it?).

This is a mostly true update, but I'd rather we change it up and just store the user payload a native object to serialize.

Thoughts?

Reply:
> just store the user payload a native object to serialize.

This is what this PR does, right?